employees/views.py

```diff
@@ -164,10 +188,13 @@ def post(self, request: HttpRequest, pk: int) -> Union[Response, HttpResponseRed
                     }
                 )
             serializer.save()
+            logger.info(f"Report with id: {report.pk} has been updated by user with id: {request.user.pk}")
         return redirect("custom-report-list")
 
 
 def delete_report(_request: HttpRequest, pk: int) -> HttpResponseRedirectBase:
+    logger.info(f"User with id: {_request.user.pk} want to delete report with id: {pk}")
     report = get_object_or_404(Report, pk=pk)
     report.delete()
+    logger.debug(f"Report with id: {report.pk} has been deleted")
```

Comment:
`report.pk` will be `None` here after deleting so there is no need to print it.

Reply:
Done

Reply:
DOne